dashboard/app/assets/stylesheets/application.scss

```diff
@@ -538,6 +539,22 @@ $default-modal-width: 640px;
       float: left;
       position: relative;
     }
+
+    .hint {
+      padding: 15px 15px 15px 40px;
+      background-color: $hint-bgcolor;
+      border-radius: 10px;
+    }
+    .hintInner {
```

Comment:
It's pretty different, but still perfectly usable without the styles. I'm just going to remove them for now:

Old:
![image](https://cloud.githubusercontent.com/assets/1070243/18687287/35ffb95c-7f34-11e6-98cd-00779e25d5e0.png)

New:
![image](https://cloud.githubusercontent.com/assets/1070243/18687308/53141fa6-7f34-11e6-98b7-289098bbfa7c.png)